src/index.ts

```diff
@@ -12,33 +13,67 @@ import { looper } from './utils/looper'
 import { parseConfig } from './utils/parse-config'
 import { validateConfig } from './utils/validate-config'
 
+import {
+  closeLog,
+  openLogfile,
+  printAllLogs,
+  flushAllLogs,
+} from './utils/history'
+
 class Monika extends Command {
-  static description = 'describe the command here'
+  static description = 'Monika command line monitoring tool'
 
   static flags = {
     // add --version flag to show CLI version
     version: flags.version({ char: 'v' }),
     help: flags.help({ char: 'h' }),
-    // flag with a value (-n, --name=VALUE)
+
     config: flags.string({
       char: 'c',
       description:
-        "JSON configuration file path e.g './config.json' (default './config.json')",
-      required: true,
+        'JSON configuration filename. If none is supplied, will look for default in current path.',
       default: './config.json',
     }),
-    // flag with no value (-f, --force)
-    force: flags.boolean({ char: 'f' }),
+
+    logs: flags.boolean({
+      char: 'l',
```

Comment:
Can we put some comment why the char is `l`?

Reply:
done.